src/fhirpath/engine/base.py

```diff
@@ -101,7 +101,7 @@ class EngineResult(object):
     body: EngineResultBody
 
     def __init__(
-        self, header: EngineResultHeader, body: EngineResultBody, scroll_id: str
+        self, header: EngineResultHeader, body: EngineResultBody, scroll_id: str = ""
```

Comment:
shouldn't the default scroll_id be None ? I am not a huge fan of empty strings for default "nullable" values

Reply:
oh, is it because URL.build does not accept None values ?

Reply:
Ironic for a gopher, right? 😉 
I'll put None as default, I don't think it'll break anything

Reply:
*str ftw